shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/ManagerClient.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.service.smartevents.shard.operator;
+
+import java.util.List;
+import java.util.function.Function;
+
+import com.redhat.service.smartevents.infra.models.dto.BridgeDTO;
+import com.redhat.service.smartevents.infra.models.dto.ProcessorDTO;
+
+import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.core.buffer.Buffer;
+import io.vertx.mutiny.ext.web.client.HttpResponse;
+
+public interface ManagerClient {
+
+    Uni<Object> fetchAndProcessBridgesToDeployOrDelete(Function<List<BridgeDTO>, Uni<Object>> handler);
```

Comment:
rename `fetchAndProcessBridgesToDeployOrDelete` to something like `fetchBridgesToDeployOrDelete`

Reply:
See https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/620